### PR TITLE
Issue 2191 Sequence Help

### PR DESCRIPTION
**Pull request by kingthorin**
Add initial helpset for Sequence Adddon.

Fixes zaproxy/zaproxy#2191

### CODE REVIEW

File: src/org/zaproxy/zap/extension/sequence/resources/help/contents/sequence.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+</HEAD>
+<BODY BGCOLOR="#ffffff">
+
+<H1>Sequence Scanner</H1>
+This add-on facilitates the recording and scanning sequences of requests. 
+In some web applications it is necessary for particular events or pages to be accessed or completed in a certain order.
+By recording a sequence and then scanning it, it is possible to ensure such flows happen in an expected manner.
+
+<H2>Creating Sequences</H2>
+There are two options for creating sequences:
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/sequence/resources/help/contents/sequence.html]: Should be more explicit saying that it should be created a script per sequence?

File: src/org/zaproxy/zap/extension/sequence/resources/help/contents/sequence.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+<br><br>
+<ol>
+<li>In either the Sites Tree or History tab select the requests you wish to have included, right click, and
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/sequence/resources/help/contents/sequence.html]: Tree could be all lower case.

File: src/org/zaproxy/zap/extension/sequence/resources/help/contents/sequence.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+There are two options for creating sequences:
+<br><br>
+<ol>
+<li>In either the Sites Tree or History tab select the requests you wish to have included, right click, and 
+use "Add To Zest Script" (either choosing to create a new script or adding to an existing Sequence script).</li>
+<li>From the main tool bar, use the "Record a New Zest Script..." button, selecting "Sequence" as the type.</li>
+</ol>
+
+<H2>Scanning</H2>
+When a new Active Scan is created there is a "Sequence" tab in which you can choose the sequence(s) to be scanned.
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/sequence/resources/help/contents/sequence.html]: Might worth saying that the tab is only shown when the advanced option is enabled.

**Comment by thc202** [comment]
Could you add a new task (to build.xml file) to generate the wiki?

**Comment by kingthorin** [comment]
Sure

**Comment by thc202** [comment]
Thank you!

**Pull request merged**